src/bridgeState.js

```diff
@@ -299,17 +304,20 @@ module.exports = class BridgeState {
   }
 
   async saveNodeState() {
-    await this.db.storeNodeState({
-      periodProposal: this.periodProposal,
-      stalePeriodProposal: this.stalePeriodProposal,
-    });
-
     if (!this.submissions.length) return;
     await this.db.storePeriods(this.submissions).then(() => {
```

Comment:
Why then here? Just set `this.submissions` after `await this.db.storePeriods(this.submissions)`:

```es6
await this.db.storePeriods(this.submissions);
this.submissions = [];
```

Reply:
probably I had brain corruption 😅